src/scaffold/template/maven/gen/maven/pom.xml

```diff
@@ -15,15 +15,16 @@
             <plugin>
                 <groupId>org.jsonschema2pojo</groupId>
                 <artifactId>jsonschema2pojo-maven-plugin</artifactId>
-                <version>0.4.37</version>
+                <version>1.0.1</version>
                 <configuration>
-                    <sourceDirectory>../../dist/model/json-schema-v3</sourceDirectory>
+                    <targetPackage>{{ customConfig.javaPackage }}</targetPackage>
+                    <sourceDirectory>${basedir}/../../dist/model/json-schema-v3</sourceDirectory>
                     <includeJsr303Annotations>true</includeJsr303Annotations>
-                    <useJodaDates>true</useJodaDates>
-                    <useJodaLocalDates>true</useJodaLocalDates>
-                    <useJodaLocalTimes>true</useJodaLocalTimes>
+                    <includeJsr305Annotations>true</includeJsr305Annotations>
                     <generateBuilders>true</generateBuilders>
                     <removeOldOutput>true</removeOldOutput>
+                    <dateTimeType>java.time.ZonedDateTime</dateTimeType>
```

Comment:
There should be a paragraph in the documentation saying that implementations need a jackson module for serializing/deserializing as written in https://github.com/joelittlejohn/jsonschema2pojo/issues/511#issuecomment-188445791